### PR TITLE
Hotfix/slider

### PR DESCRIPTION
**Pull request by kuzkov**

### CODE REVIEW

**Review by Vladissqq — COMMENTED** [review]
👍

File: src/app/slider/slider.component.html [hunk 1]
```diff
@@ -1,6 +1,22 @@
 <section class="slider">
   <div [ngStyle]="sliderContainerStyles" class="slider__container">
-    <ng-content></ng-content>
+    <div
+      #slide
+      class="slider__slide-container"
+      *ngFor="let movie of movies$ | async"
+      [ngStyle]="{ 'background-image': 'url(\'' + movie.imageUrl + '\')' }"
```
[the remark this answers is not in the document]
Comment (marinbonibon) [reply src/app/slider/slider.component.html:7]: And use template string, please

File: src/app/services/movies.service.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import movies from '../../assets/mockups/movies.json';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class MoviesService {
+  private movies: Film[] = movies;
+
+  public getRecommendedMovies(): Observable<Film[]> {
+    return of(this.movies).pipe(delay(2000));
```
[the remark this answers is not in the document]
Comment (kuzkov) [reply src/app/services/movies.service.ts:16]: yeah

File: src/app/shared/film-description/film-description.component.html [hunk 3]
```diff
@@ -2,26 +2,34 @@
...
     <span class="film-info__info-text">{{ film.country }}</span>
     <span class="film-info__info-text">
-      {{ film.year }} | {{ film.ageConstraint }}+
+      {{ film.releaseYear }} | {{ film.ageRating }}+
     </span>
-    <span class="film-info__info-rate">
-      <span>IMDb {{ film.rate }}</span>
+
+    <span
+      *ngFor="let rating of film.ratings.slice(0, 1)"
```
[the remark this answers is not in the document]
Comment (kuzkov) [reply src/app/shared/film-description/film-description.component.html:17]: okey

File: src/app/shared/film-description/film-description.component.html [hunk 4]
```diff
@@ -2,26 +2,34 @@
...
     </span>
   </div>
 
   <p class="film-info__description">{{ film.description }}</p>
 
   <div class="film-info__buttons-container">
-    <button class="film-info__button" (click)="passId()" routerLink="/video-player">
+    <button
+      class="film-info__button"
+      (click)="passId()"
```
[the remark this answers is not in the document]
Comment (kuzkov) [reply src/app/shared/film-description/film-description.component.html:30]: will be fixed

File: src/app/slider/slider.component.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (kuzkov) [reply src/app/slider/slider.component.html:7]: template strings don't work in angular html

[the remark this answers is not in the document]
Comment (kuzkov) [reply src/app/slider/slider.component.html:7]: I don't know why

[the remark this answers is not in the document]
Comment (marinbonibon) [reply src/app/slider/slider.component.html:7]: yeah, you're right, just move this object outside and use template string there)

**Pull request merged**